pandas/core/arrays/datetimelike.py

```diff
@@ -23,10 +24,13 @@
     is_timedelta64_dtype,
     is_object_dtype)
 from pandas.core.dtypes.generic import ABCSeries, ABCDataFrame, ABCIndexClass
+from pandas.core.dtypes.dtypes import DatetimeTZDtype
 
 import pandas.core.common as com
 from pandas.core.algorithms import checked_add_with_arr
 
+from .base import ExtensionOpsMixin
```

Comment:
can you do absolute imports

Reply:
Tried it, causes import-time errors

Reply:
i see, we import in __init__ all of the extension arrays which import this. hmm.